includes/AbstractTextAPI.php

```diff
@@ -62,10 +82,6 @@ public function execute() {
 		);
 	}
 
-	public function mustBePosted() {
-		return false;
```

Comment:
Why that?

Reply:
Seems to be a problem with the git diff function here - or maybe I modified indentation? It's still there just above in the code, lines 46-48